### PR TITLE
updated image and input components

### PR DESCRIPTION
**Pull request by krgarima**
This branch has input and image components.

### CODE REVIEW

**Review by sumitch29 — APPROVED** [review]
Screenshot (322)
code snippet for the last image is not visible

File: Components/Image/image.css [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+.image--container {
+  margin: 10vh;
+  width: 70vw;
+  margin-top: 15vh;
+}
+
+/* Square image */
+.square__img {
+    margin: 1rem;
+}
```
Comment (pnchinmay) [inline Components/Image/image.css:19]:
```suggestion
object-fit: cover;
}
```

File: Components/Image/image.css [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    width: 5rem;
+}
+.square__img--md {
+    height: 7.5rem;
+    width: 7.5rem;
+}
+.square__img--lg {
+    height: 10rem;
+    width: 10rem;
+}
```
Comment (pnchinmay) [inline Components/Image/image.css:31]: same here

**Comment by dusariudaykumar** [comment]
just one suggestion In input components, set input type for password !!!!!  UI looks cool and responsive

**Review by pnchinmay — CHANGES_REQUESTED** [review]

File: Components/Image/image.css (hunk 1 above)
Comment (krgarima) [reply Components/Image/image.css:19]: done

File: Components/Image/image.css (hunk 2 above)
Comment (krgarima) [reply Components/Image/image.css:31]: done.

**Comment by krgarima** [comment]
> just one suggestion In input components, set input type for password !!!!! UI looks cool and responsive

**Pull request merged**